### PR TITLE
[Feature] Be notified when the Pending > Bonded period has ended

### PR DESCRIPTION
**Pull request by mariano-aguero**
Part of #75 

### Includes
- New env var for the sendgrid template
- New helper function, that generates subscribers to send notifications
- Add new notification when the round change

### Does not include
- Logic related to telegram notification

The email should be like this:
Screenshot_20190723_182208

New version:
Screenshot_20190724_072141

### CODE REVIEW

File: server/helpers/sendDelegatorEmail.js [hunk 1]
```diff
@@ -210,9 +211,32 @@ const sendDelegatorNotificationDelegateChangeRulesEmail = async (
   return
 }
 
+const sendDelegatorNotificationBondingPeriodHasEnded = async (subscriber, delegateAddress) => {
+  try {
+    if (!subscriber.email) {
+      return
```
Comment (Agupane) [inline server/helpers/sendDelegatorEmail.js]: Could you log this case?

File: server/helpers/notification/notificationUtils.js [hunk 2]
```diff
@@ -62,28 +62,18 @@ const sendRoundNotifications = async (roundProgress, round, thresholdSendNotific
...
-    /*
-    const telegramSubscribers = await Subscriber.find({
-      frequency: 'daily',
-      activated: 1,
-      telegramChatId: { $ne: null }
-    })
-    
-     */
+    console.log(
+      `[Check-Round-Change] - Sending email notifications pending to finish bonding period has ended to delegates`
```
Comment (Agupane) [inline server/helpers/notification/notificationUtils.js]: PP: The `pending to finish bonding period` text is not so much clear. I would change it to:
`[Check-Round-Change] - Sending bonding period has ended email notifications to delegators`

Note that I changed `delegates` to `delegators`

File: server/helpers/notification/notificateDelegateUtils.js [hunk 3]
```diff
@@ -48,6 +49,64 @@ const sendEmailRewardCallNotificationToDelegates = async currentRoundInfo => {
   return subscribersToNotify
 }
 
+const sendEmailAfterBondingPeriodHasEndedNotificationToDelegates = async currentRoundInfo => {
```
Comment (Agupane) [inline server/helpers/notification/notificateDelegateUtils.js]: R: change name to: `sendEmailAfterBondingPeriodHasEndedNotificationToDelegators`

Comment (mariano-aguero) [reply server/helpers/notification/notificateDelegateUtils.js]: Done

File: server/helpers/notification/notificationUtils.js (hunk 2 above)
Comment (mariano-aguero) [reply server/helpers/notification/notificationUtils.js]: Done

File: server/helpers/sendDelegatorEmail.js (hunk 1 above)
Comment (mariano-aguero) [reply server/helpers/sendDelegatorEmail.js]: Done

**Pull request merged**